app_helper.js

```diff
@@ -86,7 +87,90 @@ async function loadMongoose() {
   await loadModels(dbConnection, options, defaultLog);
 }
 
+async function update__read_only__reports__top_search_terms(defaultLog, afterTimestamp) {
```

Comment:
These functions will need to go into separate files in a file structure like:
/api/materialized_views/